index.html

```diff
@@ -1975,42 +2069,46 @@
 
     <section id="match_pattern"> <h3>Match patterns</h3>
       <p>
-        A match pattern is a string which can contain the
+        A <a>match pattern</a> is a string which can contain the
         <code>'*'</code> wildcard character.
-        Match patterns are used primarily for matching <a>IANA media type</a>s,
-        for instance the match pattern <code>'application/*+json'</code> matches
+        It is defined by the following ABNF:
+        <pre>
+          <dfn data-lt="match pattern">match-pattern</dfn> = <a>top-level type name</a> <code>"/"</code> [ tree <code>"."</code> ] <a>subtype name</a> [ <code>"+"</code> suffix ] [ <code>";"</code> parameters ]
+          <dfn>top-level type name</dfn> = <code>"*"</code> | &lt;any char except <code>"/"</code> and <code>"*"</code>&gt;
+          <dfn>subtype name</dfn>        = <code>"*"</code> | &lt;any char except <code>"+"</code>&gt;
+        </pre>
+        A <a>match pattern</a> is used primarily for matching <a>IANA media type</a>s,
```

Comment:
It's used _only_ for matching media types.

There's also no algorithm for checking whether it matches... Say it's a glob?